CHANGELOG.adoc

```diff
@@ -12,7 +12,7 @@ The full list of changes for this release can be found in https://github.com/dev
 == 2024.11.001
 
 Release with new features and bugfixes:
-
+* https://github.com/devonfw/IDEasy/issues/587[#587]: Checks for git installation before performing git operations
```

Comment:
`2024.11.001` is already released.
You need to move this up to `2024.12.001`.
This time we should be able to include it into that release and not miss it again.

Reply:
Moved this to the right release